requirements-xpu.txt

```diff
@@ -1,8 +1,6 @@
 # Common dependencies
 -r requirements-common.txt
 
-setuptools < 70.0.0 # IPEX's torch have some dependency. to be removed.
```

Comment:
I feel we should keep this.

Reply:
https://github.com/advisories/GHSA-cx63-2mw6-8hw5